packages/common/src/api/GqlContext.tsx

```diff
@@ -1,14 +1,83 @@
 import React, { FC, useMemo, useEffect, useState, useCallback } from 'react';
 import { createContext } from 'react';
-import { GraphQLClient } from 'graphql-request';
+import {
+  BatchRequestDocument,
+  GraphQLClient,
+  RequestDocument,
+  Variables,
+} from 'graphql-request';
+import { AuthError } from '../authentication/AuthContext';
+import { LocalStorage } from '../localStorage';
 
-export const createGql = (url: string): { client: GraphQLClient } => {
-  const client = new GraphQLClient(url, { credentials: 'include' });
+class GQLClient extends GraphQLClient {
```

Comment:
extending it solely to avoid typing problems elsewhere. the fun of using a codegen 🙄